### DISCUSSION TITLE
Only two weeks to go!

### DISCUSSION THREAD

**Discussion by maxbannach** [esa/SpOC3#11] [Announcements]
Dear SpOC Competitors!

Time is running out! Only two weeks to go for the final push: to find
a larger torso, improve the interferometric measurements, or to
finally build the Enterprise using programmable cubes.

Keep in mind that this year, SpOC is implementing a *2-out-of-3
scoring scheme* for the global scoring. That means participants who
submit only solutions to one of the three challenges may miss out! But
do not worry; there is still plenty of room on the scoreboards! 

The ACT is rooting for you in these final two weeks! Best of luck, SpOC Competitors!

**Comment by dietmarwo** [comment]
What happens if the two best 2-out-of-3 scores are equal? Does then the 3-out-of-3 score sum count or do we have two winners then?

**Reply by maxbannach** [reply]
Ties on the global leaderboard will result in shared placements, i.e., two winners / second places, etc.

**Reply by dietmarwo** [reply]
Thanks for the clarification. Regarding the "plenty of room on the Scoreboards" may be the issue discussed in https://github.com/esa/SpOC3/discussions/6 played a role. Spacekangaroos posted an optimal solution at https://optimise.esa.int/challenge/spoc-3-interferometric-mission/p/tiny-observatory-in-dro realizing that they loose 5 points at  interferometric even if they would create the best solution for all three difficulty levels. This could be demotivating. 
Beside that issue interferometric is an exciting task, easily accessible since heyoka does an excellent job here. You can start using the objective function out of the box. My suggestion is to avoid this kind of issue next year.
